### PR TITLE
Onboarding document

### PR DESCRIPTION
**Pull request by sandre35**
Linked issue: https://github.com/tournesol-app/tournesol/issues/207

### CODE REVIEW

**Review by GresilleSiffle — COMMENTED** [review]
Thanks @sandre35 !

There are very useful insights on several topics that will greatly improve the Tournesol documentation. Actually, now that we have more contributors, I feel the need of a well described contribution process to smooth the development, and make the contributors as autonomous as possible. In this context, your documentation is more than welcome : )

Even if this `ONBOARDING.md` document is good, I would not recommend to merge it as is for now. We can still save it in a file like `docs/onboarding.md` to unlock this pull request if you don't want to dive into a big documentation and translation effort. On my side, and if you agree, I'm ready to break this document into several pieces to improve the current documentation scattered in the repository.  

Here are my suggestions.

**(1)** 

The content mixes several topics like the architecture overview, some installation instructions, few tips and a contributing guide.

I would have split all of this in their respective documents.

For instance, the automatic installation with Docker is already documented in the `dev-env`, and the manual installation process is also documented in the `backend` and `frontend` folders. To avoid having to edit several files when the installation process evolves, I recommend to not include installation instructions in an on-boarding document, but rather include an invitation to check the existing documentation with a Mardown link.

The contribution part could go into a standard `CONTRIBUTING.md` file. This file could contain the complete contribution process, including the technical steps to follow before submitting a PR ( like running the code quality tools ).

The overview could go into `CONTRIBUTING.md`, or even the main `README.md` so that new comers don't have to jump from a file to another to gather those kind of generic information. It will also reduce the maintenance cost, as editing the theses file is more easier than remembering there is a architecture documentation somewhere in the documentation in `docs`.

**(2)** The current file is in french, and we will translate it in english.

File: ONBOARDING.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+- Pour corriger le style du code de manière automatique, il est possible d’activer un plugin selon l’IDE. Sinon on la commande `yarn lint:fix` permet dans tous les cas de corriger le code de manière automatique.
+
+# Fonctionnement de MR/projet
+
+Le suivi du projet et des tâches en cours se fait sur ce [board](https://github.com/tournesol-app/tournesol/projects/9)
+
+Il est possible de créer des idées qui nous semblent pertinentes en éditant une issue et en la plaçant dans la colonne `Backlog and ideas`.
+
+La core-team décide par la suite des tâches à prioriser et dépose les tickets correspondants dans la colonne `Ready`. On peut s’assigner sur un ticket après avoir discuté avec [Adrien][Adrien], [Romain][Romain]. ou [Louis][Louis].
```
Comment (GresilleSiffle) [inline ONBOARDING.md:104]: Nice!

If we translate this part, I'd recommend to add an invitation for the contributors to ask questions is the issue is not that clear, and maybe to agree with the team before starting to work if the task requires to choose between several options (as all options could have different consequences, more or less impacting).

File: ONBOARDING.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+Quand votre PR vous semble dans un bon état pour être mergé, demandez la revue d’[Adrien][Adrien], [Romain][Romain] et [Louis][Louis] (au moins). L’opération de merge ne peut être effectuée que si au moins une personne approuve votre PR.
+
+Un changement dans le back doit impliquer des tests pour contrôler le comportement attendu, ainsi que les possibles comportements qui doivent être évités.
+
+Côté front, une modification peut aussi s’accompagner de tests pour le component dédié.
+
+La création de test end-to-end n’est pas systématique et il est bon de voir avec [Adrien][Adrien] et [Romain][Romain] pour savoir l’utilité d’un nouveau test.
+
+Pour se tenir au courant des avancées du projet, des bugs, rejoignez le [discord][tournesol-discord-join]
```
Comment (sandre35) [inline ONBOARDING.md:116]: This part is going to be translated in another document called `Contributing.md`

**Comment by amatissart** [comment]
Most of these insights have been included in #466, thanks @sandre35  and @GresilleSiffle.
I am also closing #207. 

Don't hesitate to suggest additional changes in README or CONTRIBUTING if necessary.

**Pull request closed without merging**